### PR TITLE
fix(#1639): web-components module file update

### PR DESCRIPTION
**Pull request by amar-ABgov**

### CODE REVIEW

**Review by ArakTaiRoth — APPROVED** [review]

**Review by chrisolsen — APPROVED** [review]

File: libs/web-components/package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
   "private": true,
```
Comment (chrisolsen) [inline libs/web-components/package.json:14]: Can you also remove this line (#14)

**Review by chrisolsen — CHANGES_REQUESTED** [review]

**Review by chrisolsen — APPROVED** [review]

**Pull request merged**